packages/docs/docs/provider/set-up.md

```diff
@@ -0,0 +1,71 @@
+# Set Up
+
+This section covers the initial setup process for your Medplum system, including how to add practitioners and import existing data through the Medplum App.
+
+<br />
+
+--- 
+
+## **Adding Practitioners (via Medplum App)**
+
+Setting up practitioners is one of the first steps in configuring your Medplum system and will need to be completed using the Medplum App (app.medplum.com). Practitioners represent healthcare providers, clinicians, and other staff members who will be using the system.
+
+### **Prerequisites**
+
+* Administrative access to the Medplum App  
+* Practitioner information including names, credentials, and contact details  
+* Valid email addresses for each practitioner (required for system access)
+
+### **How to Add Practitioners**
+
+1. **Access the Practitioners Section**  
```

Comment:
This is incorrect, the correct link is here: https://app.medplum.com/admin/users
Consider linking to the User Management guide: https://www.medplum.com/docs/auth/user-management-guide